pkg/federate/cluster_informer.go

```diff
@@ -0,0 +1,25 @@
+package federate
+
+import (
+	"k8s.io/client-go/rest"
+)
+
+// ClusterEventHandler handles federated cluster lifecycle event notifications.
+type ClusterEventHandler interface {
+	// OnAdd is called when a cluster is added. The given 'kubeConfig' can be used to access
```

Comment:
Hope we're not going for a design where broker *needs* to access kube API endpoints of joined clusters. This requirement of Kubefed is causing problems with private cluters, so would prefer to avoid this.

Reply:
The broker implementation won't use the interfaces in this file at all as it has no need to be informed of cluster lifecycle. And neither will lighthouse need this interface. We could just remove this file altogether.